src/sketch.js

```diff
@@ -1,21 +1,28 @@
 /**
  * p5.js setup.
- * 
+ *
  * Returns:
  *  undefined.
  */
-function setup() 
-{
+let song;
 
+function preload() {
+  song = loadSound('assets/audio/da_beat-nr1.mp3');
+}
+
+function setup()
+{
+  createCanvas(800, 200);
+  song.loop();
```

Comment:
Please document this line:

```suggestion

  // loop theme song
  song.loop();
```